br/br-pitr-manual.md

```diff
@@ -392,5 +392,6 @@ Restore KV Files <--------------------------------------------------------------
 
 > **Note:**
 >
+> - When you restore the cluster for the first time, it is necessary to specify the full snapshot data, otherwise some data in new created table may be incorrect due to rewriting Table ID rules.
```

Comment:
```suggestion
> - When you restore the cluster for the first time, you must specify the full snapshot data. Otherwise, some data in newly created table might be incorrect due to rewriting Table ID rules.
```